bin/autoupdate

```diff
@@ -1,5 +1,20 @@
 #!/usr/local/bin/python-odoo-shell
+import os
+
 
 # Note: ``module_auto_update`` must be installed in Odoo for this to work.
-env["base.module.upgrade"].upgrade_module()
+try:
+    env["ir.module.module"].upgrade_changed_checksum
+except AttributeError:
+    env["base.module.upgrade"].upgrade_module()
+else:
+    # Disable deprecated stuff
+    env["ir.config_param"].set_param(
+        "module_auto_update.enable_deprecated",
+        "0",
```

Comment:
I'm not sure if we should do this here or leave existing implantations to decide. That wouldn't harm us, I just wanted to publish this to hear others' opinions.

Reply:
Well, I think it's not too problematic to leave this way.